geomstats/geometry/poincare_ball.py

```diff
@@ -227,32 +229,73 @@ def mobius_add(self, point_a, point_b):
         return mobius_add
 
     @geomstats.vectorization.decorator(['else', 'vector', 'vector'])
-    def dist(self, point_a, point_b):
-        """Compute the geodesic distance between two points.
+    def dist(self, points_a, points_b):
```

Comment:
We chose the singular form to be the convention of parameters' names and docstring: see methods' signature and docstrings in the codebase.